src/envo/envo-edit.owl

```diff
@@ -40276,6 +40277,20 @@ AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_06105124>
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105124> "coastal anthropogenic litter"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105124> <http://purl.obolibrary.org/obo/ENVO_03500005>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105125> (plastic production)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://en.wikipedia.org/wiki/Plastics_industry>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105125> "A manufacturing process which has plastic as its specified output."@en)
```

Comment:
@iwilkie this works, however, it scopes this manufacturing process to only that which produces plastic as its only specified output. 

If this is what you mean, that's fine, but if plastic can be one of multiple possible outputs, I'd say:

>A manufacturing process which includes plastic as one of its specified outputs.